src/disp.c

```diff
@@ -58,7 +59,7 @@ void display_help(void)
 void display_verbose(char *data[], char *data_names[])
 {
 	int i = 0;
-	for (i = 0; i < 16; i++)
+	for (i = 0; i < DETECTED_ARR_LEN; i++)
```

Comment:
I think when the array length is 17 this should either be `i < 17` or `i <= 16`.